src/libwalletqt/TransactionHistory.cpp

```diff
@@ -143,7 +148,11 @@ bool TransactionHistory::TransactionHistory::locked() const
 TransactionHistory::TransactionHistory(Monero::TransactionHistory *pimpl, QObject *parent)
     : QObject(parent), m_pimpl(pimpl), m_minutesToUnlock(0), m_locked(false)
 {
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+    m_firstDateTime = QDate(2014, 4, 18).startOfDay();
+#else
     m_firstDateTime  = QDateTime(QDate(2014, 4, 18)); // the genesis block
+#endif
```

Comment:
Could you post the warning?

Reply:
```
../../Qt/5.15.0/clang_64/lib/QtCore.framework/Headers/qdatetime.h:296:5: note: 'QDateTime' has been explicitly marked deprecated here
    QT_DEPRECATED_X("Use QDate::startOfDay()") explicit QDateTime(const QDate &);
    ^
```

Reply:
I see. Wonder they don't mention this in the docs.